Lighthouse Camera/UI/Main/PreviewView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  PreviewView.swift
+//  Lighthouse Camera
+//
+//  Created by Punam Dahiya on 1/27/17.
+//  Copyright © 2017 Lighthouse. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+protocol PreviewViewDelegate: class {
+  func displayPreviewSession()
+}
+
+
+class PreviewView: UIView {
+
+  var delegate: PreviewViewDelegate?
+
+  required init?(coder aDecoder:NSCoder) {
+    super.init(coder: aDecoder)
+    self.startPreviewSession()
+  }
+
+  var videoPreviewLayer: AVCaptureVideoPreviewLayer {
+    return layer as! AVCaptureVideoPreviewLayer
+  }
+
+  var session: AVCaptureSession? {
+    get {
+      return videoPreviewLayer.session
+    }
+    set {
+      videoPreviewLayer.session = newValue
+    }
+  }
+
+  // MARK: UIView
+
+  override class var layerClass: AnyClass {
+    return AVCaptureVideoPreviewLayer.self
+  }
+
+  // Accessibility
+    
+  override var isAccessibilityElement: Bool {
+    get {
+      return true
+    }
+    set {
+      self.isAccessibilityElement = true
+    }
+  }
+    
+  override var accessibilityLabel: String? {
+    get {
+      return NSLocalizedString("Peview Screen", comment: "")
+    }
+    set {
+      self.accessibilityLabel = NSLocalizedString("Peview Screen", comment: "")
+    }
+  }
+    
+  func startPreviewSession() {
+    // Establish Capture Session
+    let captureSession = AVCaptureSession()
+
+    self.session = captureSession
+
+    let sessionOutput = AVCapturePhotoOutput()
+    let backCamera = AVCaptureDevice.defaultDevice(withMediaType: AVMediaTypeVideo)
+
+    var error: NSError?
+    var input: AVCaptureDeviceInput!
+    do {
```

Comment:
That looks weird. I guess you do that just to avoid capturing too many errors, right? Could you add a comment?